components/table/concurrent_queue.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
```

Comment:
in such implementation it is not a concurrent_queue_t and it is not very suitable for our stage of competitiveness to execution